Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -232,6 +233,20 @@ private void InitializeNotifyIcon()
             };
         }
 
+        private void CheckFirstLaunch()
+        {
+            if (_settings.FirstLaunch)
+            {
+                _settings.FirstLaunch = false;
+                _viewModel.Save();
```

Comment:
i dont think this is the right save call:
![image](https://user-images.githubusercontent.com/26427004/144582281-1f7a53f4-18d3-4c45-af8d-1b855dfe1e62.png)

it's not saving the setting, so if you terminate after flow is launched, it will display the welcome screen again on next launch. is this intended?

Reply:
I thought it should be saved in that location.  I knew the problem that it could not be saved this timing . but I didn't know how to call save(). so I commented it above.

>Currently, FirstLaunch is applied only when the flow is terminated. First of all, I don't think there will be a problem, but it would be nice to have a code that safely stores settings inside CheckFirstLaunch().

I wanted anyone to work on it, but I thought it was okay because no one responded. 

Anyway, taoo checked and added the save code.